contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,60 @@
+Enforcement team
+===============
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, the enforcement team
+pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. Our main priority is to ensure the safety of our community members.
+The second goal is to help educate the community as a whole to be aware of the CoC
+and how to help implement its spirit throughout the community. In case these goals
+conflict, we will prioritize safety of community members over all other goals.
+
+If you think there is or has been a violation to the code of conduct please contact
+enforcement team or of you prefer contact only individual members of the enforcement team.
+
+Members
+-------
+
+Here are all the members of the Code of Conduct enforcement team. You can contact
+any of them by sending an email or tweet. You could also contact all of them at once
+by emailing **coc@symfony.com**.
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, susciit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+About the Enforcement Team
+--------------------------
+
+The :doc:`Symfony core team </contributing/code/core_team>` appoints enforcement
+team with candidates they see fit. The enforcement team will consists of at least
+be 3 persons. The team should be representing as many demographics as possible,
+ideally from different employers and not in a private partnership with Symfony
+and/or SensioLabs.
```

Comment:
I don't understand why we should have a specific case for Symfony/SensioLabs.

Reply:
agreed ... I think something went wrong there any way during some "refactoring"

how about:
`The team should be representing as many demographics as possible, ideally from different employers and not in a private partnership`

Reply:
in Drupal Dries afaik does appoint the enforcement team. imho it would be better to stread this decision a bit wider, but it of course makes things simpler.

that being said, giving Fabien, or the maintainer team, the final say on all enforcement is imho not a good idea. getting enforcement right requires investing the necessary time not only in each specific case but also in educating yourself on the topic, attending relevant trainings etc.